keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -443,107 +448,173 @@ General prep:
 Prep for brownfield:
 + admin creates `Bucket` instances representing backend buckets.
 
+### Finalizers
+The following finalizers are added by COSI to orchestrate deletion of the various COSI resources. Of note is that a finalizer is added to the two user created instances (BR and BAR) so that these intances remain available until the deletion of the associated resources (Bucket and BA) and the backend bucket have completed.
+
++ _objectstorage.k8s.io/user-protect_, added to BRs and BARs.
```

Comment:
Does "user-protect" also indicate user creates BR or BAR?  If so, please add to the comment.

Reply:
Finalize name has been changed, one for BR and different for BAR